tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderTests.cs

```diff
@@ -375,5 +375,48 @@ public void CalculateCostPerYear_OneOff_OrderItemCostTypeRecurring_ReturnsZero()
 
             order.CalculateCostPerYear(CostType.OneOff).Should().Be(0);
         }
+
+        [Test]
+        public void CalculateTotalOwnershipCost_RecurringAndOneOff_CalculatesTotalOwnership()
```

Comment:
Very minor but could the method name be changed to what it returns i.e. ReturnsTotalOwershipCost? This would then follow the pattern of the other methods in this file.